examples/js/postprocessing/RenderPass.js

```diff
@@ -39,7 +39,7 @@ THREE.RenderPass.prototype = Object.assign( Object.create( THREE.Pass.prototype
 
 		}
 
-		renderer.render( this.scene, this.camera, readBuffer, this.clear );
+		renderer.render( this.scene, this.camera, this.renderToScreen ? null : readBuffer, this.clear );
```

Comment:
Oh wait. I guess this pass doesn't need a `readBuffer` and it renders directly to it.
It's confusing though...